Adding Github templates (#3052)
KratosMultiphysics/Kratos: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve

---

**Scope of the bug**

> Where in Kratos is this bug happening. Ex

Reviewer: ```suggestion
Where in Kratos is this bug happening. E.g.,
```

- Compilation
- KratosCore
- FooApplication

> **Description the bug**

Reviewer: ```suggestion
**Description of the bug**
```

> A clear and concise description of what the bug is.

Reviewer: ```suggestion
A clear and concise description of the bug.
```

**To Reproduce**

> Steps to reproduce the bug (if possible). Add here any custom example files as well in case they are relevant.

Reviewer: ```suggestion
Steps to reproduce the bug (if possible). Add any custom example files here if they are relevant.
```

**Expected behavior**
A clear and concise description of what you expected to happen.

**Environment**

> Please specify your operating system, kratos branch and python version and any other library versions that you consider relevant.

Reviewer: ```suggestion
Please specify your operating system, kratos branch, python version and any other library versions that you consider relevant.
```

> Ex:

Reviewer: ```suggestion
E.g.:
```
